[DOCFIX] Update cluster install docs (#11050)
Alluxio/alluxio: docs/en/deploy/Running-Alluxio-On-a-Cluster.md

## Overview

This section describes the basic setup to run Alluxio with a single master in a cluster.
This is the simplest way to deploy Alluxio on a cluster, meanwhile this single master may also 
become the single point of failure (SPOF) in an Alluxio cluster.

> If that machine or process became unavailable, the cluster as a whole would be unavailable.

Reviewer: ```suggestion
If that machine or process becomes unavailable, the cluster as a whole would become unavailable.
```

You use `become` in the sentence before, so it seems weird to switch tenses

To deploy Alluxio in production, we highly recommend running Alluxio masters in
[High Availability]({{ '/en/deploy/Running-Alluxio-On-a-HA-Cluster.html' | relativize_url }}) mode.

## Prerequisites

* To deploy a Alluxio cluster, first [download](https://alluxio.io/download) the pre-compiled
  Alluxio binary file, extract the tarball with the below command, and copy the extracted 
  directory to all nodes (including nodes running masters and workers).
  
  ```console
  $ tar -xvzpf alluxio-{{site.ALLUXIO_VERSION_STRING}}-bin.tar.gz
  ```
  
* Enable SSH login without password from master node to worker nodes.
  You can add a public SSH key for the host into `~/.ssh/authorized_keys`.

>   See [this tutorial] (http://www.linuxproblem.org/art_9.html) for more details.

Reviewer: ```suggestion
  See [this tutorial](http://www.linuxproblem.org/art_9.html) for more details.
```

* TCP traffic across all nodes is allowed.
  For basic functionality make sure RPC port (default :19998) is open on all nodes.

> * Allow `sudo` privilege for the OS user that Alluxio will be running as.

Reviewer: why is this a requirement?

Author: Further down we mention starting Alluxio with `./bin/alluxio-start.sh all SudoMount`. For consistency, I think we can put this as a requirement. Alternatively, I can add more to the start section to explain how to avoid `sudo`.

Reviewer: I don't think it's a hard requirement especially because it depends on configuration and mounting (NVMe?). `sudo` is only ever required to actually make the mount which is separate from Alluxio functionality. I think it is ok to put that `sudo` is required if users want our scripts to mount the workers' storage directories for them

Author: done

```

- The first property `alluxio.master.hostname` sets the hostname of the single master node.
  Examples include
  `alluxio.master.hostname=1.2.3.4` or `alluxio.master.hostname=node1.a.com`.
- The second property `alluxio.master.mount.table.root.ufs` sets to the URI of the under store to
  mount to the Alluxio root.
  This shared shared storage system must be accessible by the master node and all worker nodes.
  Examples include `alluxio.master.mount.table.root.ufs=hdfs://1.2.3.4:9000/alluxio/root/`, or 
  `alluxio.master.mount.table.root.ufs=s3://bucket/dir/`.

Next, copy configuration files to all the other Alluxio nodes.
By adding the IP addresses or hostnames of all the worker nodes to the `conf/workers` file, an 

> operator can make use of in-built utilities to copy configurations to remote nodes such as below.

Reviewer: ```suggestion
operator can make use of built-in utilities to copy configurations to remote nodes such as below.
```

On any master node, format the Alluxio journal with the following command:

```console
$ ./bin/alluxio formatMaster
```

Formatting the journal will delete all metadata from Alluxio. However, the data in under storage will be untouched.

### Add/Remove Workers Dynamically

Adding a worker to an Alluxio cluster dynamically is as simple as starting a new Alluxio worker

> process, with the appropriate configuration.

Reviewer: ```suggestion
process with the appropriate configuration.
```

In most cases, the new worker's configuration should be the same as all the other workers'
configuration.
Run the following command on the new worker to add it to the cluster.

```console
$ ./bin/alluxio-start.sh worker SudoMount # starts the local worker
```

Once the worker is started, it will register itself with the Alluxio master, and become part of the
Alluxio cluster.

Removing a worker is as simple as stopping the worker process.

```console
$ ./bin/alluxio-stop.sh worker # stops the local worker
```

Once the worker is stopped,the master will flag the worker as lost after a predetermined timeout 
value (configured by master parameter `alluxio.master.worker.timeout`).

> The master will consider the worker as "lost", and no longer consider it as part of the cluster.

Reviewer: re-word to not use "consider" twice?

Author: done
